triggers/Library/LaunchDaemons/com.github.patchoo-trigger-patchoo.plist

```diff
@@ -6,7 +6,7 @@
 	<string>com.github.patchoo-trigger-patchoo</string>
 	<key>ProgramArguments</key>
 	<array>
-		<string>/usr/sbin/jamf</string>
+		<string>/usr/localbin/jamf</string>
```

Comment:
Sorry about that commit #3 should address?

On Wed, Mar 23, 2016 at 11:36 PM, Lachlan Stewart <notifications@github.com>
wrote:

> In triggers/Library/LaunchDaemons/com.github.patchoo-trigger-patchoo.plist
> <https://github.com/patchoo/patchoo/pull/56#discussion_r57271178>:
>
> > @@ -6,7 +6,7 @@
> >  	<string>com.github.patchoo-trigger-patchoo</string>
> >  	<key>ProgramArguments</key>
> >  	<array>
> > -		<string>/usr/sbin/jamf</string>
> > +		<string>/usr/localbin/jamf</string>
>
> Thanks heaps... but a weeeee typo right here!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/patchoo/patchoo/pull/56/files/7daa8921b941e195c9e1c832a01e733d7fa716ba..1102a8dcedb2ce3c49d8a50f8badcee96eede846#r57271178>
>